### PR TITLE
Display rockets - Lists render

### LINKED ISSUE #13: [4pt] Display rockets - Lists render
- Use `useSelector()` Redux Hook to select the state slices and render lists of rockets in corresponding routes. i.e.:
```javascript
// get rockets data from the store
const rockets = useSelector(state => state.rockets);
```

- You can style the whole application "by hand" or you could use [React Bootstrap](https://react-bootstrap.github.io/), a UI library that could speed up the process. This is a popular library and working with its components would be good practice.
- Render a list of rockets (as per design). For the image of a rocket use the first image in the array of `flickr_images`.

### PR DESCRIPTION
**Pull request by Emmanuelaaron**
In this milestone, the following requirements have been met:
- [x] Use `useSelector()` Redux Hook to select the state slices and render lists of rockets in corresponding routes.
- [x] We chose to style the application using React Bootstrap.
- [x] Render a list of rockets (as per design). For the image of a rocket, we used the first image in the array of `flickr_images`.

### CODE REVIEW

File: src/components/Rockets.js [hunk 1]
```diff
@@ -14,7 +16,27 @@ const Rockets = () => {
...
+    <div className="">
+      {rockets.map((rocket) => (
+        <Card
+          bg="ligh"
+          key={rocket.id}
+          text="dark"
+          className="m-2 row flex-row align-items-center p-4"
+        >
+          <div className="col-sm-12 col-md-3 text-center ">
+            <Card.Img variant="left" src={rocket.flickr_images[0]} className="w-100 rocket-image" height="200" style={{ maxWidth: '25vw', maxHeight: '25vw' }} />
```
Comment (enionsouza) [inline src/components/Rockets.js]:
```suggestion
            <Card.Img variant="left" src={rocket.flickr_images[0]} style={{ width: '250px', height: '250px', objectFit: 'cover' }} />
```
Nice work, @Emmanuelaaron! What do you think about this suggestion?

**Review by enionsouza — CHANGES_REQUESTED** [review]
Your code is working very well, but I think we could make some changes to the image renderings...
@Emmanuelaaron, can you please take a look at the suggestion for the presentation of the images?

File: src/components/Rockets.js (hunk 1 above)
Comment (Emmanuelaaron) [reply src/components/Rockets.js]: Thanks, @enionsouza for taking note of the image rendering. I'm implementing the suggested updates

**Review by enionsouza — APPROVED** [review]
Nice job, @Emmanuelaaron! LGTM!

**Review by mariordgez — APPROVED** [review]
Looks good to me, nice job!

**Pull request merged**